domain/src/main/java/org/oppia/android/domain/oppialogger/LoggingIdentifierController.kt

```diff
@@ -121,8 +147,21 @@ class LoggingIdentifierController @Inject constructor(
     sessionId.value = computeSessionId()
   }
 
+  /**
+   * Regenerates [appSessionId] and notifies the data provider.
+   *
+   * The [appSessionId] is generally updated when:
+   *  1. The app is opened.
+   *  2. The app is destroyed by the Android system.
+   */
+  fun updateAppSessionId() {
```

Comment:
Why is this needed? It seems like we shouldn't ever need to explicitly updated the session ID since it autoinitializes upon app open.

Reply:
I realize that now. I will remove this.